modules/charon-core/pom.xml

```diff
@@ -84,6 +91,6 @@
                     </instructions>
                 </configuration>
             </plugin>
-        </plugins>
+    </plugins>
```

Comment:
There should be a formatting issue if you look at the indentation of opening plugin tag